src/Pages/yechanhong/Login/Login.js

```diff
@@ -1,10 +1,152 @@
 import React, { Component } from "react";
-export class LoginHong extends Component {
+import "./Login.scss";
+import { Link } from "react-router-dom";
+
+class LoginHong extends Component {
+  constructor() {
+    super();
+    this.state = {
+      loginVaildation: false,
+      loginValue: "",
+      passwordValue: "",
+    };
+  }
+
+  handleIdValue = (e) => {
+    this.setState({
+      loginValue: e.target.value,
+    });
+  };
+
+  handlePasswordValue = (e) => {
+    this.setState({
+      passwordValue: e.target.value,
+    });
+  };
+
+  handleLoginValue = () => {
+    if (
+      this.state.loginValue.includes("@") &&
+      this.state.loginValue.length >= 5 &&
+      this.state.passwordValue.length >= 5
+    ) {
+      this.setState({
+        loginVaildation: true,
+      });
+    } else {
+      this.setState({
+        loginVaildation: false,
+      });
+    }
```

Comment:
이렇게 하면 더 간결해집니다!
```suggestion
   const loginVaildation =  this.state.loginValue.includes("@") &&
      this.state.loginValue.length >= 5 &&
      this.state.passwordValue.length >= 5
      this.setState({
        loginVaildation: loginVaildation,
      });
```
그리고 객체에서는 key와 value가 똑같으면 더 줄일 수 있어요!
```
 this.setState({
        loginVaildation,
      });
```

Reply:
관희님께서 review 해주신 대로 변경했지만 이렇게 변경해도 가능한 이유(?) 원리(?)를 잘 모르겠어요...이 부분에 대해서 다시 설명해주실 수 있으신가요??ㅠㅠ

Reply:
원리 보다는 자바스크립트 ES6에서 이게 가능하게끔 만든 "문법"입니다!